src/containers/my-cooperations/cooperation-details/CooperationDetails.tsx

```diff
@@ -132,6 +152,17 @@ const CooperationDetails = () => {
     return cooperationContent
   }
 
+  const isCooperationClosingRequestSend =
+    response.needAction === userRole &&
+    response.status === StatusEnum.RequestToClose
+
+  const acceptClosingProccess = !isClosed && (
+    <AcceptCooperationClosing
+      onAccept={handleCooperationCloseAccept}
+      user={response.initiator.firstName ?? 'user'}
```

Comment:
You should not use just `initiator`, because receiver also can request closing cooperation.